Doc/faq/windows.rst

```diff
@@ -339,7 +339,6 @@ Sometimes, when you download the documentation package to a Windows machine
 using a web browser, the file extension of the saved file ends up being .EXE.
 This is a mistake; the extension should be .TGZ.
 
-Simply rename the downloaded file to have the .TGZ extension, and WinZip will be
-able to handle it.  (If your copy of WinZip doesn't, get a newer one from
-https://www.winzip.com.)
-
+Simply rename the downloaded file to have the .TGZ extension, and your
+favorite file compression manager should be able to handle it.  If not look
+for a newer version that does it/this.
```

Comment:
I think we should continue to mention something specific (possibly still winzip), though it would be reasonable to add "or your favorite archive program".  It is so hard to know what software is "good software" and what is virus-ware, and where to get safe copies of the good ones, in the windows world.  (I just went through this myself, and ended up downloading 7-zip, but frankly getting it, unsigned, from sourceforge made me nervous even though I was installing it in a VM).